client/rest-high-level/src/main/java/org/elasticsearch/client/ml/job/results/CategoryDefinition.java

```diff
@@ -140,6 +163,10 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
         if (grokPattern != null) {
             builder.field(GROK_PATTERN.getPreferredName(), grokPattern);
         }
+        builder.field(NUM_MATCHES.getPreferredName(), numMatches);
+        if (preferredToCategories != null && preferredToCategories.isEmpty() == false) {
```

Comment:
I would feel better if `preferredToCategories.isEmpty() == false` was in parentheses, just in case ;)